Dockerfile

```diff
@@ -29,5 +29,4 @@ RUN git clone https://gitlab.gnome.org/GNOME/glib.git --depth=1 && \
     (cd /graphene && \
         meson setup builddir --prefix=/usr --buildtype release -Dintrospection=disabled -Dtests=false -Dinstalled_tests=false && \
         meson install -C builddir) && \
-    rm -rf /glib /cairo /harfbuzz /pango /gdk-pixbuf /graphene
-
+    rm -rf /glib /cairo /harfbuzz /pango /gdk-pixbuf /graphene
```

Comment:
The change on this file is unneeded, please revert it

Reply:
The idea was to trigger a rebuild of the image with this, but I guess that can be done manually too?